src/backend/CassandraBackend.cpp

```diff
@@ -1087,14 +1077,18 @@ CassandraBackend::open(bool readOnly)
            << ", result: " << rc << ", " << cass_error_desc(rc);
         throw std::runtime_error(ss.str());
     }
-    if (getInt("max_write_requests_outstanding"))
-        maxWriteRequestsOutstanding = *getInt("max_write_requests_outstanding");
 
-    if (getInt("max_read_requests_outstanding"))
-        maxReadRequestsOutstanding = *getInt("max_read_requests_outstanding");
+    if (auto val = config_.maybe_value<int>("max_write_requests_outstanding");
+        val)
+        maxWriteRequestsOutstanding = *val;
+
+    if (auto val = config_.maybe_value<int>("max_read_requests_outstanding");
+        val)
+        maxReadRequestsOutstanding = *val;
+
+    if (auto val = config_.maybe_value<int>("sync_interval"); val)
+        syncInterval_ = *val;
```

Comment:
Why this syntax versus:
```
maxWriteRequestsOutstanding = config_.value_or<int>("max_write_requests_outstanding",maxWriteRequestsOutstanding);
```
I believe this is the syntax you used elsewhere.

Reply:
one less assignment but an extra branch so maybe you are right. it's not all that important tbh in this context - i can change to the other option